frontend/src/component/admin/groups/GroupForm/GroupForm.tsx

```diff
@@ -108,18 +137,48 @@ export const GroupForm: FC<IGroupForm> = ({
                     data-testid={UG_DESC_ID}
                 />
                 <ConditionallyRender
-                    condition={Boolean(uiConfig.flags.syncSSOGroups)}
+                    condition={Boolean(
+                        uiConfig.flags.syncSSOGroups && !isOss()
+                    )}
```

Comment:
Only need to wrap `uiConfig.flags.syncSSOGroups` in the `Boolean()`, not `isOss()`